modules/libstemmer/2.2.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,22 @@
+load("@rules_cc//cc:defs.bzl", "cc_library")
+
+cc_library(
+    name = "libstemmer",
```

Comment:
Nit: You could call this just `stemmer` to avoid output files with duplicated `lib` prefix and then add a public alias called `libstemmer` for convenience.